receiver/mongodbreceiver/metadata.yaml

```diff
@@ -238,3 +256,39 @@ metrics:
       aggregation: cumulative
       monotonic: false
     attributes: []
+  mongodb.locks.acquire_count:
```

Comment:
```suggestion
  mongodb.lock.acquire.count:
```

Reply:
Updated the metric name as per suggestion